src/components/secrets/Secret.tsx

```diff
@@ -610,6 +598,25 @@ export const SecretForm: React.FC<SecretFormProps> = function (props) {
         setSecretDataYaml(secretYaml)
     }
 
+    const closeDeleteModal = (): void => {
+        setShowDeleteModal(false)
+    }
+
+    const setShowDelete = (): void => {
+        setShowDeleteModal(true)
+    }
+
+    const renderDeleteCIModal = () => {
+        return (
+            <DeleteDialog
+                title={`Delete '${props.name}' ?`}
+                description={`Are you sure you want to delete this secret ?`}
```

Comment:
Please remove {``} and use direct string

Reply:
Added variable in description